### ISSUE TITLE
Add an actual title option.

### ISSUE THREAD

**Issue by Darkwolf1515** [nextcloud/notes#584]
I would prefer to have a separate title bar, rather than just taking the first line, doesn't feel right to me.

**Comment by stefan-niedermann** [comment]
I vote against it. In my opinion this results in a higher cognitive load when trying to just scratch down *anything* - a random number, a phrase, a poem, an date, ...

I have to think and categorize about the topic first, because i would do this instinktively as soon as i see the title field (no matter whether or not it is mandatory).

The title *can* be changed and set manually already. I can't see a problem which would be solved by always displaying an esit field which animates users to fill it.

**Comment by korelstar** [comment]
Like @stefan-niedermann said, you can change the title manually. Please use the three-dots menu in the app-navigation for this purpose.

If this is not sufficient, please be more precise and provide screenshots and/or mock-ups, since I'm not sure what you feature request is really about.

**Comment by Darkwolf1515** [comment]
![image](https://user-images.githubusercontent.com/65789901/90438488-9027e680-e0c3-11ea-9b61-0fb4343f1d1a.png)

**Comment by Darkwolf1515** [comment]
![image](https://user-images.githubusercontent.com/65789901/90438532-a2098980-e0c3-11ea-94b0-c3d8bbd92cb1.png)

**Comment by adroslice** [comment]
The default behaviour of the first line being the title can still be preserved, just have the title bar update with that first line. Sure it's slightly redundant, but I find the workflow of using a context menu to change the seperately from the notes first line excruciatingly frustrating. On the same note, working with categories is also a pain, since that menu card on the right isn't visible by default and requires ANOTHER context menu to pop out. And then it doesn't even stay open when I switch to another note!
Note: I'm conflating three-dot-menus and context menus here, but even a proper right click context menu that works anywhere on the note would be better.

Here is my suggestion to vastly improve the experience:
Keep the details card, but make it stay expanded upon switching notes, perhaps even persistently for the user, but per-session is okay too. Then, simply make the title that is shown there able to be edited directly.

This way, those who choose to keep it as minimal as possible still can work with just the note, but those who want easy access to "advanced" features still have it.

**Comment by daudtivan** [comment]
Agree with @adroslice that advanced options should be there for those that would like to do "something else" with that note.

For those that simply want to make a quick note, the title at the 1st line followed by the actual note (like this G Keep snapshot) is sufficient.
> ![image](https://user-images.githubusercontent.com/65789901/90438488-9027e680-e0c3-11ea-9b61-0fb4343f1d1a.png)

**Comment by user239393** [comment]
I agree. It is super annoing when you have to press a special combination of keys to get to a result. This is not inutiv. For Nextcloud Note you always have to press the checkbox button that the Note gets a title. Instead the user should be able to see a field for the title that he can change it how he wants.
